normalize_repos/normalize_repos.py

```diff
@@ -26,10 +29,16 @@ def get_cc_repos(github):
     cc = get_cc_organization(github)
     return cc.get_repos()
 
+def check_project_status(repo):
```

Comment:
A better name would be `is_engineering_project` (or similar) as Boolean functions should have the interrogative form.

Reply:
Got it. Done.